app/src/lib/git/rev-parse.ts

```diff
@@ -52,6 +52,32 @@ export async function getTopLevelWorkingDirectory(
   return Path.resolve(path, relativePath)
 }
 
+export async function checkIfRepositoryIsBare(
+  path: string
+): Promise<boolean | null> {
+  let result
+
+  try {
+    result = await git(
+      ['rev-parse', '--is-bare-repository'],
+      path,
+      'checkIfRepositoryIsBare'
+    )
+  } catch (e) {
+    if (e.message.includes('not a git repository')) {
+      return null
```

Comment:
Are you able to add some documentation to the header of this function, so we can understand when we should expect `null` versus a boolean versus an error?